### PR TITLE
cod 175 generate remix app

### PR DESCRIPTION
**Pull request by lundinmathias**
- **chore(repo): remove existing web app**
- **feat(web): generate new remix web app**

### CODE REVIEW

File: apps/web-e2e/tsconfig.json [hunk 1]
```diff
@@ -3,8 +3,8 @@
   "compilerOptions": {
     "allowJs": true,
     "outDir": "../../dist/out-tsc",
-    "module": "commonjs",
-    "sourceMap": false
+    "sourceMap": false,
```
Comment (hakalb) [inline apps/web-e2e/tsconfig.json:6]: No effect

File: apps/web/project.json [hunk 2]
```diff
@@ -1,17 +1,9 @@
 {
   "name": "web",
   "$schema": "../../node_modules/nx/schemas/project-schema.json",
-  "sourceRoot": "apps/web/src",
+  "sourceRoot": "apps/web",
   "projectType": "application",
-  "tags": ["scope:web", "type:app"],
+  "tags": [],
```
Comment (hakalb) [inline apps/web/project.json:6]: Why?

**Review by hakalb — COMMENTED** [review]
Just minors, merge when you want to

File: apps/web/project.json (hunk 2 above)
Comment (lundinmathias) [reply apps/web/project.json:6]: Missed tags, will add them.

**Pull request merged**